recipes/gdbm/all/conanfile.py

```diff
@@ -96,7 +97,11 @@ def package(self):
             autotools = self._configure_autotools()
             autotools.install()
         os.unlink(os.path.join(self.package_folder, "lib", "libgdbm.la"))
+        compat_la = os.path.join(self.package_folder, "lib",
+                                 "libgdbm_compat.la")
+        if (os.path.exists(compat_la)):
+            os.unlink(compat_la)
         tools.rmdir(os.path.join(self.package_folder, "share"))
 
     def package_info(self):
-        self.cpp_info.libs = ["gdbm"]
+        self.cpp_info.libs = tools.collect_libs(self)
```

Comment:
This will allow collecting `libgdbm_compat.a` if it was requested and produced via option `libgdbm_compat=True`